packages/x-date-pickers/src/DateCalendar/PickersSlideTransition.tsx

```diff
@@ -1,16 +1,19 @@
 import * as React from 'react';
 import clsx from 'clsx';
-import { styled, useTheme, useThemeProps } from '@mui/material/styles';
+import { useTheme } from '@mui/material/styles';
```

Comment:
Can't `useTheme` be statically extracted? 🤔 
If it can, maybe it should be imported from `internals/zero-styled` as well?

Reply:
I asked it some time ago and it it's used for transitions that it's fine without code migration 
https://mui-org.slack.com/archives/C0583QU37EF/p1717492417498109?thread_ts=1717492276.312499&cid=C0583QU37EF